spec/mongoid/config_spec.rb

```diff
@@ -224,6 +224,79 @@
     end
   end
 
+  context 'async_query_executor option' do
```

Comment:
I think you can just use `it_behaves_like "a config option"` like the tests below this one... Might just be a lot cleaner.

Reply:
As I understood, `a config option` assumes that an option can be either `true` or `false`. Here the option is a symbol.

Reply:
Ah yes, looking at those examples you are correct.